Source/Data/Excel/xl_ipre.txt

```diff
@@ -0,0 +1,85 @@
+PLName(s)	PLPower(i)	PLParam1(i)	PLParam2(i)	PLMinLvl(i)	PLIType(i)	PLGOE(i)	PLDouble(i)	PLOk(i)	PLMinVal(i)	PLMaxVal(i)	PLMultVal(i)
```

Comment:
The extension should be `tsv`, then it will display nicely on GitHub as well

Reply:
I've never heard of TSV, and the extension doesnt appear to be recognized by any of the tabbed text editors I use (though newer excel versions might). Diablo 2 also used .txt for the excel files, and I'm heavily basing the style off that as it naturally evolved from D1.

Reply:
TSV is extremely ubiquitous! Likely supported even before Excel 97.

Reply:
https://en.m.wikipedia.org/wiki/Tab-separated_values

Reply:
Afaik TSV is even older then CSV. Even if it's not what D2 and possibly D1 used it's a small change for a big benefit in readability and usability.

Reply:
I wouldn't say it benefits usability, since it's an extra nuance of having to register the file extension or select "all file types" in the drop down box. The only benefit is to look pretty while you view it on github, and IMHO that's not important. Whether or not you guys change it is on you, I'll be keeping it this way on my main project.

Reply:
Thats more of an editor epecific issue, what editor do you use?

Reply:
If you drag and drop a tabbed .txt file onto excel it will automatically detect it. If you open in program it asks you for the delimiter type. When exporting .XLS files as tabbed text, it exports them with a .txt extension.

I typically use AFJ sheet for editing them, which was a tabbed text tool designed for diablo 2.
![Capture](https://user-images.githubusercontent.com/15209402/76557206-197a2380-6469-11ea-8e0e-327ed1024229.PNG)